RecoLocalCalo/Configuration/python/ecalLocalRecoSequence_cff.py

```diff
@@ -4,6 +4,7 @@
 from RecoLocalCalo.EcalRecProducers.ecalRecHitTPGConditions_cff import *
 
 # ECAL reconstruction
+from RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitPhase2_cff import *
```

Comment:
I suggest not to change this piece of configuration, but modify the object directly in the file
 RecoLocalCalo/EcalRecProducers/test/testEcalUncalibRecHitPhase2WeightsProducer.py